mat64/format.go

```diff
@@ -11,13 +11,14 @@ import (
 
```

Comment:
I haven't interacted with this function before. At first glance, the signature is pretty daunting. My comments are made with that in mind.